mypy/plugin.py

```diff
@@ -81,6 +117,12 @@ def get_method_signature_hook(self, fullname: str) -> Optional[MethodSignatureHo
     def get_method_hook(self, fullname: str) -> Optional[MethodHook]:
         return None
 
+    def get_attribute_hook(self, fullname: str) -> Optional[AttributeHook]:
```

Comment:
I'm not sure I like the proliferation of different callback types. Even though these can be statically checked, it seems a pain for users to keep them apart. (Plus, how easy is it really for users to type-check their plugins? There are no stubs for mypy.)